tests/conftest.py

```diff
@@ -894,6 +945,19 @@ def multi_pvc_factory_class(
     )
 
 
+@pytest.fixture(scope='session')
+def multi_pvc_factory_session(
+    storageclass_factory_session,
+    project_factory_session,
+    pvc_factory_session
+):
+    return multi_pvc_factory_fixture(
+        storageclass_factory_session,
+        project_factory_session,
+        pvc_factory_session
```

Comment:
@fbalak just to be sure, we will be creating all kinds of PVCs.. right ? RWO and RWX and attach them to PODs accordingly ?

Reply:
Right, but for exact configuration I will probably need your help.